package.json

```diff
@@ -12,10 +12,13 @@
     "postinstall": "cd packages/react-instantsearch && npm install",
     "doctoc": "doctoc --maxlevel 1 README.md CONTRIBUTING.md",
     "test": "(cd packages/react-instantsearch && npm test) && npm run lint && NODE_ENV=production DOCS_MOUNT_POINT=/instantsearch.js/react/ DOCS_DIST=docs/react/ npm run docs:build && NODE_ENV=production npm run build",
-    "dev": "cd packages/react-instantsearch && npm run dev"
+    "dev": "cd packages/react-instantsearch && npm run dev",
+    "storybook": "start-storybook -p 6006",
```

Comment:
I think I would be ok with having react/story endpoint.

Being in dev a live storybook env and in prod our full blown storybook for people to get inspired/play (well it might just be the same actually).

What do you think?

If it really feels like a very good fast env for you, then let's do it